public/app/percona/dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -40,6 +40,7 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({ values, form })
   const [loadingAllocatedResources, setLoadingAllocatedResources] = useState(false);
   const [expectedResources, setExpectedResources] = useState<DBClusterExpectedResources>();
   const [loadingExpectedResources, setLoadingExpectedResources] = useState(false);
+  const mounted = { current: true };
```

Comment:
is this supposed to mimic `useRef` ?
I think the value won't hold in case of re-render, as it happens with that hook.

Reply:
It's mentioned in the description of the PR: "not using a ref here because the value needs to be set to true every time the component renders".

Don't want it to hold but to be set to true on every render. Its ok because after the component unmounts there are no more renders (obviously) and this is variable is set to false, so when the request finishes it doesn't schedule another one.

If a ref was used the value would be false after the first rerender and then it would not trigger the request when it should.

Reply:
Oh, sorry, missed the description. All good then 👍